_data/toc.yaml

```diff
@@ -22,43 +22,49 @@ guides:
   section:
   - path: /engine/installation/
     title: Install Docker
-  - sectiontitle: Docker Edge
+  - sectiontitle: Docker EE
     section:
-    - path: /edge/
-      title: Overview
-    - path: /edge/engine/reference/commandline/docker/
-      title: Docker Edge CLI reference
-      nosync: true
-    - path: /engine/userguide/eng-image/multistage-build/
-      title: Use multi-stage builds (Edge only)
-  - path: /docker-for-mac/install/
-    title: Docker for Mac
-  - path: /docker-for-windows/install/
-    title: Docker for Windows
-  - path: /docker-ee-for-windows/install/
-    title: Docker EE for Windows Server
-  - sectiontitle: Docker for Linux
+    - path: /docker-ee-for-windows/install/
+      title: Docker EE for Windows Server
+    - path: /engine/installation/linux/docker-ee/ubuntu/
+      title: Docker EE for Ubuntu
+    - path: /engine/installation/linux/docker-ee/rhel/
+      title: Docker EE for RHEL
+    - path: /engine/installation/linux/docker-ee/centos/
+      title: Docker EE for CentOS
+    - path: /engine/installation/linux/docker-ee/oracle/
+      title: Docker EE for Oracle Linux
+    - path: /engine/installation/linux/docker-ee/sles/
+      title: Docker EE for SLES
+  - sectiontitle: Docker CE
     section:
-    - path: /engine/installation/linux/ubuntu/
+    - path: /docker-for-mac/install/
+      title: Docker for Mac
+    - path: /docker-for-windows/install/
+      title: Docker for Windows
+    - path: /engine/installation/linux/docker-ce/ubuntu/
       title: Ubuntu
-    - path: /engine/installation/linux/rhel/
-      title: Red Hat Enterprise Linux
-    - path: /engine/installation/linux/centos/
+    - path: /engine/installation/linux/docker-ce/debian/
+      title: Debian
```

Comment:
Should we change these titles to `Docker CE for Debian` (and so on)?

![img_0703](https://cloud.githubusercontent.com/assets/1804568/26376688/51a8d714-4005-11e7-9e5c-24763ee9f0f4.PNG)